### PR TITLE
WIP: add endpoint to check permission

### PR DESCRIPTION
**Pull request by healfy**

### CODE REVIEW

File: W3ChampionsIdentificationService.Tests/TestsAppConfig.cs [hunk 1]
```diff
@@ -4,8 +4,10 @@ namespace W3ChampionsIdentificationService.Tests
 {
     public class TestsAppConfig : IAppConfig
     {
-        public string MongoConnectionString => "mongodb://157.90.1.251:3712"; // "mongodb://localhost:27017";
```
Comment (Cepheid-UK) [inline W3ChampionsIdentificationService.Tests/TestsAppConfig.cs:7]: this is a publically available test DB, its fine to keep this IP here, you can just swap the comments

File: W3ChampionsIdentificationService/ServiceVerification/B2BVerificationService.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            ValidateLifetime = true,
+            ValidateIssuerSigningKey = true,
+            ValidIssuer = _issuer,
+            ValidAudience = _audience,
+            IssuerSigningKey = _key
+        };
+
+        try
+        {
+            tokenHandler.ValidateToken(token, validationParameters, out _);
```
Comment (Cepheid-UK) [inline W3ChampionsIdentificationService/ServiceVerification/B2BVerificationService.cs:35]: it's a minor quibble, but I dont like that we are calling a library function "ValidateToken" inside a function called "ValidateToken"

File: W3ChampionsIdentificationService/RolesAndPermissions/Controllers/PermissionsController.cs [hunk 3]
```diff
@@ -64,5 +65,12 @@ public async Task<IActionResult> Update([FromBody] Permission permission)
             }
             return Ok();
         }
+        [HttpPost("checkPermission")]
+        [B2BVerification]
```
Comment (Cepheid-UK) [inline W3ChampionsIdentificationService/RolesAndPermissions/Controllers/PermissionsController.cs:69]: If we're adding this functionality, we should also add it to more endpoints (if that's to come later, no problem)

Comment (healfy) [reply W3ChampionsIdentificationService/RolesAndPermissions/Controllers/PermissionsController.cs:69]: yeah, I was planing to add later, it's just MVP to introduce new approach

File: W3ChampionsIdentificationService.Tests/TestsAppConfig.cs (hunk 1 above)
Comment (healfy) [reply W3ChampionsIdentificationService.Tests/TestsAppConfig.cs:7]: sure, but it didn't work for me, and I thought that it's not public url, I would return it back

File: W3ChampionsIdentificationService/ServiceVerification/B2BVerificationService.cs (hunk 2 above)
Comment (healfy) [reply W3ChampionsIdentificationService/ServiceVerification/B2BVerificationService.cs:35]: fixed

**Pull request closed without merging**